packages/cactus-plugin-ledger-connector-corda/src/main/json/openapi.json

```diff
@@ -455,16 +455,16 @@
                 "type": "object",
                 "required": [
                     "success",
-                    "transactionId",
-                    "callOutput"
```

Comment:
@elenaizaguirre Left another comment in the service code, I'd love it if we could somehow keep the callOutput mandatory (had an idea in that other comment as well). Sorry for throwing around ideas instead of having concrete steps, the local test environment is not working on my machine right now for some reason.

Reply:
Done!